plugins/modules/vcenter_vmtemplate_libraryitems.py

```diff
@@ -348,24 +348,6 @@
     memory:
       hot_add_enabled: true
       size_MiB: 1024
-    disks:
```

Comment:
Also fix the return value according to the new VM parameter (just one default disk and no nics), please.
Should be something similar:
```
            "cpu": {
                "cores_per_socket": 1,
                "count": 1
            },
            "disks": {
                "2000": {
                    "capacity": 17179869184,
                    "disk_storage": {
                        "datastore": "datastore-3036"
                    }
                }
            },
            "guest_OS": "RHEL_7_64",
            "memory": {
                "size_MiB": 1024
            },
            "nics": {},
            "vm_home_storage": {
                "datastore": "datastore-3036"
            },
            "vm_template": "vm-77122"
```